examples/concurrent/main.go

```diff
@@ -40,16 +40,19 @@ func run(contextDir, dockerfile, imageName, remoteBuildURI, volumeSource, volume
 		RemoteURI:  remoteBuildURI,
 		StdIN:      input,
 	}
+	containerOpts := &provision.ContainerOptions{}
+	if volumeSource != "" {
+		if volumeDestination == "" {
+			volumeDestination = volumeSource
```

Comment:
Now i see you can't do because this line https://github.com/nuveo/gofn/blob/container_options/examples/concurrent/main.go#L48 is only in first if